Content.Server/Forensics/Components/FingerprintMaskComponent.cs

```diff
@@ -6,5 +6,8 @@ namespace Content.Server.Forensics
     /// </summary>
     [RegisterComponent]
     public sealed partial class FingerprintMaskComponent : Component
-    {}
+    {
+        [DataField]
```

Comment:
I'm not sure but there is probably a better way to do this? I assume this is done so the inheritance isn't all freaky but its super sussy 😭... Maybe ask on the discord? They might know a trick.

Reply:
Yeah I honestly don't know 🤷... Feels like no matter what way you do it its gonna be ugly 😆